fe/fe-core/src/main/java/com/starrocks/external/iceberg/cost/IcebergTableStatisticCalculator.java

```diff
@@ -35,6 +36,7 @@
 import static com.google.common.collect.ImmutableList.toImmutableList;
 import static java.util.stream.Collectors.toSet;
 
+// TODO @caneGuy add cache for statistics, currently getting stats from iceberg metafiles is not time consuming
```

Comment:
Yes, please support cache. I'm quite worried generating plan process will timeout once the scan file list is long. Plus, is there any possibility that same table stats has been computed for multiple times. For example, we have scan nodde on the same table but with different predicates.

Reply:
how about add cache in an other pr? @dirtysalt

Reply:
https://github.com/StarRocks/starrocks/issues/3497 i created a issue for this, and i will submit a pr next week